phpcs.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
```

Comment:
Don't need this file if not going to use phpcs.

Reply:
fixed